tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -167,158 +2235,305 @@ def test_get_metrics_and_expectations(
                 "profiler_config": {
                     "name": "test_volume_data_assistant",
                     "config_version": 1.0,
-                    "variables": {},
+                    "variables": {"bootstrap_random_seed": 43792},
```

Comment:
Instead of hardcoding, why don't we reference the constant?

Reply:
Please apply throughout - this will help refactoring efforts if we ever have to change the seed.

Reply:
@cdkini Doing it now -- great catch! Thanks! /cc @NathanFarmer